### PR TITLE
some refactoring

### PR DESCRIPTION
**Pull request by ustamansangat**

### CODE REVIEW

File: calculatorApp.py [hunk 1]
```diff
@@ -1,103 +1,88 @@
 import tkinter as tk
 from tkinter import *
-    
-class calc(Tk):
-    def set_window_size_and_title(self,x,y, title_name):
-        self.x = x
-        self.y = y
-        self.title_name = title_name
+
+def getLambda(func, text):
```
Comment (ustamansangat) [inline calculatorApp.py:4]: This function "binds" the argument `text` to the function `func` and returns a lambda.
This fixes your issue with binding to a loop variable, which would always use the last value set in the loop.

File: calculatorApp.py [hunk 2]
```diff
@@ -1,103 +1,88 @@
...
+    return lambda: func(text)
+
+class Calc(Tk):
+    def __init__(self):
```
Comment (ustamansangat) [inline calculatorApp.py:8]: using a constructor

File: calculatorApp.py [hunk 3]
```diff
@@ -1,103 +1,88 @@
...
+
+        for i, op in enumerate(['/', '*', '-', '+']):
+            button = tk.Button(self, text=op, width=8, height=H, command=getLambda(self._op_pressed, op))
+            button.grid(row = 1 + i, column = 3, padx = 1, pady = 1)
+
+        equals = tk.Button(self, text='=', width=18, height=H, command= lambda: self._equals_pressed())
+        equals.grid(row=4, column=1, columnspan=2, padx=1, pady=1)
+
+    def _render_textfield(self):
+        self.inputs = tk.Entry(self, width=14)
```
Comment (ustamansangat) [inline calculatorApp.py:31]: fyi for me textfield still didn't render but I didn't invest more time in figuring out which version of Python and which version of TKinter to use.
I highly advise you to use conda or pipenv.

**Comment by intondiscott** [comment]
Thank you very much, I appreciate all of this :) I should've updated to let
you know I now use miniconda for my environment. Now I put my gui stuff in
one environment with gui packages and put others in their place too. Now
from there what do I do when uploading to github? Do I include in Readme of
python version and packages used and their versions?

**Comment by ustamansangat** [comment]
my familiarity with conda / miniconda is a bit rusty
but requirements.txt or environment.yml is what it supports, google it

**Comment by intondiscott** [comment]
Cool thx! Will take a look...

**Pull request merged**